GitHubData/Models/CoreModel.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="CoreModel.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace GitHubData.Models;
+
+/// <summary>
+/// Core information about the rate limits of an authorized
+/// or unauthorized request.
```

Comment:
Refactor code docs to below:

```suggestion
/// Core information about the rate limits of an authorized or unauthorized request.
```